### PR TITLE
FUSETOOLS2-1204 - Adjust test to have them workign with VS Code 1.58+

### PR DESCRIPTION
**Pull request by apupier**
- disable trust store for unit test
- upgrade to latest vscode-extension tester to have ui test working

### CODE REVIEW

**Comment by apupier** [comment]
on Linux, the UI tests are no more working:

image

it is when setting the absolute path of the file in the inputbox, it is overriden by /bin....

currently, we are usign a workaround avoid  ntaive dialog. i tried by setting back usage of native dialog and there is trust store popup breaking the flow!

XML DSL support Camel URI code completion Code completion is working for additional endpoint options (the part after  )

File: test Fixture with speci@l chars/.vscode/settings.json [hunk 1]
```diff
@@ -1,10 +1,11 @@
...
+		"/home/apupier/.config/Code/User/globalStorage/redhat.vscode-camelk/java-dependencies-3.0.0/*.jar",
+		"/home/apupier/.config/Code/User/globalStorage/redhat.vscode-camelk/java-dependencies-3.9.0/dependencies/*.jar"
+	],
     "didact.registered": [
         "{\"name\":\"Didact Demo\",\"category\":\"Didact\",\"sourceUri\":\"/home/apupier/.vscode/extensions/redhat.vscode-didact-0.1.6/demo/didact-demo.didact.md\"}",
         "{\"name\":\"Create a New Didact Tutorial Extension\",\"category\":\"Didact\",\"sourceUri\":\"/home/apupier/.vscode/extensions/redhat.vscode-didact-0.1.6/create_extension/create-new-tutorial-with-extension.didact.md\"}"
 	]
-}
\ No newline at end of file
+}
```
Comment (apupier) [inline test Fixture with speci@l chars/.vscode/settings.json]: need to remove these modifications

File: src/JavaManager.ts [hunk 2]
```diff
@@ -27,4 +27,4 @@ export function retrieveJavaExecutable() {
 	} else {
 		return 'java';
 	}
-}
\ No newline at end of file
+}
```
Comment (apupier) [inline src/JavaManager.ts:30]: to be removed

**Comment by apupier** [comment]
3 tests failed on CI...

```
  1) XML DSL support
       Camel URI code completion
         Open "camel-context.xml" file inside Editor View:
     Error: No editor with title 'camel-context.xml' available
      at EditorGroup.<anonymous> (node_modules/monaco-page-objects/out/components/editor/EditorView.js:288:23)
      at Generator.next (<anonymous>)
      at fulfilled (node_modules/monaco-page-objects/out/components/editor/EditorView.js:5:58)
      at runMicrotasks (<anonymous>)
      at processTicksAndRejections (internal/process/task_queues.js:95:5)

  2) XML DSL support
       Camel URI code completion
         Code completion is working for component schemes (the part before the ":"):
     NoSuchElementError: no such element: Unable to locate element: {"method":"css selector","selector":".editor-instance"}
  (Session info: chrome=89.0.4389.128)
  (Driver info: chromedriver=89.0.4389.23 (61b08ee2c50024bab004e48d2b1b083cdbdac579-refs/branch-heads/4389@{#294}),platform=Linux 4.15.0-1106-aws x86_64)
      at Object.checkLegacyResponse (node_modules/selenium-webdriver/lib/error.js:546:15)
      at parseHttpResponse (node_modules/selenium-webdriver/lib/http.js:509:13)
      at /home/circleci/camel-lsp-client-vscode/node_modules/selenium-webdriver/lib/http.js:441:30
      at runMicrotasks (<anonymous>)
      at processTicksAndRejections (internal/process/task_queues.js:95:5)
  From: Task: WebElement.findElement(By(css selector, .editor-instance))
      at Driver.schedule (node_modules/selenium-webdriver/lib/webdriver.js:807:17)
      at EditorView.schedule_ (node_modules/selenium-webdriver/lib/webdriver.js:2010:25)
      at EditorView.findElement (node_modules/selenium-webdriver/lib/webdriver.js:2057:17)
      at new AbstractElement (node_modules/monaco-page-objects/out/components/AbstractElement.js:39:48)
      at new ElementWithContexMenu (node_modules/monaco-page-objects/out/components/ElementWithContextMenu.js:19:1)
      at new Editor (node_modules/monaco-page-objects/out/components/editor/Editor.js:20:9)
      at new TextEditor (node_modules/monaco-page-objects/out/components/editor/TextEditor.js:24:1)
      at Context.<anonymous> (out/src/ui-test/xml_completion_support_test.js:50:32)
      at Generator.next (<anonymous>)
      at /home/circleci/camel-lsp-client-vscode/out/src/ui-test/xml_completion_support_test.js:8:71
  From: Task: WebElement.findElement(By(css selector, .inputarea))
      at Driver.schedule (node_modules/selenium-webdriver/lib/webdriver.js:807:17)
      at TextEditor.schedule_ (node_modules/selenium-webdriver/lib/webdriver.js:2010:25)
      at TextEditor.findElement (node_modules/selenium-webdriver/lib/webdriver.js:2057:17)
      at TextEditor.<anonymous> (node_modules/monaco-page-objects/out/components/editor/TextEditor.js:111:42)
      at Generator.next (<anonymous>)
      at /home/circleci/camel-lsp-client-vscode/node_modules/monaco-page-objects/out/components/editor/TextEditor.js:8:71
      at new Promise (<anonymous>)
      at __awaiter (node_modules/monaco-page-objects/out/components/editor/TextEditor.js:4:12)
      at TextEditor.getText (node_modules/monaco-page-objects/out/components/editor/TextEditor.js:110:16)
      at TextEditor.<anonymous> (node_modules/monaco-page-objects/out/components/editor/TextEditor.js:317:39)

  3) XML DSL support
       Camel URI code completion
         Code completion is working for endpoint options (the part after the "?"):
     NoSuchElementError: no such element: Unable to locate element: {"method":"css selector","selector":".editor-instance"}
  (Session info: chrome=89.0.4389.128)
  (Driver info: chromedriver=89.0.4389.23 (61b08ee2c50024bab004e48d2b1b083cdbdac579-refs/branch-heads/4389@{#294}),platform=Linux 4.15.0-1106-aws x86_64)
      at Object.checkLegacyResponse (node_modules/selenium-webdriver/lib/error.js:546:15)
      at parseHttpResponse (node_modules/selenium-webdriver/lib/http.js:509:13)
      at /home/circleci/camel-lsp-client-vscode/node_modules/selenium-webdriver/lib/http.js:441:30
      at runMicrotasks (<anonymous>)
      at processTicksAndRejections (internal/process/task_queues.js:95:5)
  From: Task: WebElement.findElement(By(css selector, .editor-instance))
      at Driver.schedule (node_modules/selenium-webdriver/lib/webdriver.js:807:17)
      at EditorView.schedule_ (node_modules/selenium-webdriver/lib/webdriver.js:2010:25)
      at EditorView.findElement (node_modules/selenium-webdriver/lib/webdriver.js:2057:17)
      at new AbstractElement (node_modules/monaco-page-objects/out/components/AbstractElement.js:39:48)
      at new ElementWithContexMenu (node_modules/monaco-page-objects/out/components/ElementWithContextMenu.js:19:1)
      at new Editor (node_modules/monaco-page-objects/out/components/editor/Editor.js:20:9)
      at new TextEditor (node_modules/monaco-page-objects/out/components/editor/TextEditor.js:24:1)
      at Context.<anonymous> (out/src/ui-test/xml_completion_support_test.js:63:32)
      at Generator.next (<anonymous>)
      at /home/circleci/camel-lsp-client-vscode/out/src/ui-test/xml_completion_support_test.js:8:71
  From: Task: WebElement.findElement(By(css selector, .inputarea))
      at Driver.schedule (node_modules/selenium-webdriver/lib/webdriver.js:807:17)
      at TextEditor.schedule_ (node_modules/selenium-webdriver/lib/webdriver.js:2010:25)
      at TextEditor.findElement (node_modules/selenium-webdriver/lib/webdriver.js:2057:17)
      at TextEditor.<anonymous> (node_modules/monaco-page-objects/out/components/editor/TextEditor.js:111:42)
      at Generator.next (<anonymous>)
      at /home/circleci/camel-lsp-client-vscode/node_modules/monaco-page-objects/out/components/editor/TextEditor.js:8:71
      at new Promise (<anonymous>)
      at __awaiter (node_modules/monaco-page-objects/out/components/editor/TextEditor.js:4:12)
      at TextEditor.getText (node_modules/monaco-page-objects/out/components/editor/TextEditor.js:110:16)
      at TextEditor.<anonymous> (node_modules/monaco-page-objects/out/components/editor/TextEditor.js:317:39)

  4) XML DSL support
       Camel URI code completion
         Code completion is working for additional endpoint options (the part after "&"):
     NoSuchElementError: no such element: Unable to locate element: {"method":"css selector","selector":".editor-instance"}
  (Session info: chrome=89.0.4389.128)
  (Driver info: chromedriver=89.0.4389.23 (61b08ee2c50024bab004e48d2b1b083cdbdac579-refs/branch-heads/4389@{#294}),platform=Linux 4.15.0-1106-aws x86_64)
      at Object.checkLegacyResponse (node_modules/selenium-webdriver/lib/error.js:546:15)
      at parseHttpResponse (node_modules/selenium-webdriver/lib/http.js:509:13)
      at /home/circleci/camel-lsp-client-vscode/node_modules/selenium-webdriver/lib/http.js:441:30
      at runMicrotasks (<anonymous>)
      at processTicksAndRejections (internal/process/task_queues.js:95:5)
  From: Task: WebElement.findElement(By(css selector, .editor-instance))
      at Driver.schedule (node_modules/selenium-webdriver/lib/webdriver.js:807:17)
      at EditorView.schedule_ (node_modules/selenium-webdriver/lib/webdriver.js:2010:25)
      at EditorView.findElement (node_modules/selenium-webdriver/lib/webdriver.js:2057:17)
      at new AbstractElement (node_modules/monaco-page-objects/out/components/AbstractElement.js:39:48)
      at new ElementWithContexMenu (node_modules/monaco-page-objects/out/components/ElementWithContextMenu.js:19:1)
      at new Editor (node_modules/monaco-page-objects/out/components/editor/Editor.js:20:9)
      at new TextEditor (node_modules/monaco-page-objects/out/components/editor/TextEditor.js:24:1)
      at Context.<anonymous> (out/src/ui-test/xml_completion_support_test.js:76:32)
      at Generator.next (<anonymous>)
      at /home/circleci/camel-lsp-client-vscode/out/src/ui-test/xml_completion_support_test.js:8:71
  From: Task: WebElement.findElement(By(css selector, .inputarea))
      at Driver.schedule (node_modules/selenium-webdriver/lib/webdriver.js:807:17)
      at TextEditor.schedule_ (node_modules/selenium-webdriver/lib/webdriver.js:2010:25)
      at TextEditor.findElement (node_modules/selenium-webdriver/lib/webdriver.js:2057:17)
      at TextEditor.<anonymous> (node_modules/monaco-page-objects/out/components/editor/TextEditor.js:111:42)
      at Generator.next (<anonymous>)
      at /home/circleci/camel-lsp-client-vscode/node_modules/monaco-page-objects/out/components/editor/TextEditor.js:8:71
      at new Promise (<anonymous>)
      at __awaiter (node_modules/monaco-page-objects/out/components/editor/TextEditor.js:4:12)
      at TextEditor.getText (node_modules/monaco-page-objects/out/components/editor/TextEditor.js:110:16)
      at TextEditor.<anonymous> (node_modules/monaco-page-objects/out/components/editor/TextEditor.js:317:39)
```

**Review by lhein — APPROVED** [review]

**Comment by apupier** [comment]
was a flaky test. puttign back ready fo rreview flag

**Comment by apupier** [comment]
@lhein ah sorry, I "requested your review" although you already approved it.

**Review by lhein — APPROVED** [review]
I can approve it once more if that makes you happy :)

**Review by bfitzpat — APPROVED** [review]

File: test Fixture with speci@l chars/.vscode/settings.json [hunk 3]
```diff
@@ -2,7 +2,7 @@
     "java.project.referencedLibraries": [
         "lib/**/*.jar",
         "/home/apupier/.config/Code/User/globalStorage/redhat.vscode-camelk/java-dependencies-3.0.0/*.jar"
```
Comment (djelinek) [inline test Fixture with speci@l chars/.vscode/settings.json:4]: seems to be some local setup? do we need this in master branch?

Comment (apupier) [reply test Fixture with speci@l chars/.vscode/settings.json:4]: Effectively a local setup and seems no need but not modified in this PR.
Planning to remove it in another PR.
There is also the didact.registered.
Maybe th ewhoel file shoudl be removed. I need to check when master will be back to green.

**Review by djelinek — APPROVED** [review]

File: test Fixture with speci@l chars/.vscode/settings.json (hunk 3 above)
Comment (apupier) [reply test Fixture with speci@l chars/.vscode/settings.json:4]: see https://github.com/camel-tooling/camel-lsp-client-vscode/pull/688

**Pull request merged**